domain/port/state/watcher.go

```diff
@@ -23,11 +24,11 @@ func (st *State) WatchOpenedPortsTable() string {
 // InitialWatchOpenedPortsStatement returns the query to load the initial event
 // for the WatchOpenedPorts watcher
 func (st *State) InitialWatchOpenedPortsStatement() string {
-	return "SELECT uuid FROM machine"
+	return "SELECT name FROM machine"
 }
 
 // GetMachinesForEndpoints returns a slice of machine UUIDs that host the provided endpoints.
-func (st *State) GetMachinesForEndpoints(ctx context.Context, eps []string) ([]string, error) {
+func (st *State) GetMachinesForEndpoints(ctx context.Context, eps []string) ([]coremachine.Name, error) {
```

Comment:
I wish this returned `map[string]....` then you can see which machines are associated with which endpoint.